src/types/discord.ts

```diff
@@ -325,3 +327,35 @@ export interface IntegrationDeleteEvent {
   /** id of the bot/OAuth2 application for this discord integration */
   "application_id"?: string;
 }
+
+export interface InviteCreateEvent {
+  /** the channel the invite is for */
+  "channnel_id": string;
+  /** the unique invite code */
+  code: string;
+  /** the time at which tehi invite was created */
+  "created_at": string;
+  /** the guild of the invite */
+  "guild_id": string;
```

Comment:
```suggestion
  "guild_id"?: string;
```
Source: https://discord.com/developers/docs/topics/gateway#invite-create